docs/trulens_eval/api/feedback.md

```diff
@@ -16,6 +16,8 @@ Any Custom App Wrapper: [TruCustomApp](https://www.trulens.org/trulens_eval/api/
 
 ::: trulens_eval.trulens_eval.feedback.provider.litellm
 
+::: trulens_eval.trulens_eval.feedback.provider.bedrock
+
```

Comment:
nit: also add the base class here, also maybe rename to llm_provider instead of base for clarity